charts/pgadmin4/README.md

```diff
@@ -50,6 +50,7 @@ The command removes nearly all the Kubernetes components associated with the cha
 | `image.tag` | Docker image tag | `""` |
 | `image.pullPolicy` | Docker image pull policy | `IfNotPresent` |
 | `annotations` | Deployment Annotations | `{}` |
+| `command` | Deployment command override | [["/entrypoint.sh"]](https://github.com/postgres/pgadmin4/blob/master/Dockerfile#L206) |
```

Comment:
you are correct that the default is entrypoint, but I rather have this set as empty in the README.md example.